src/validations/noFakeLists.js

```diff
@@ -0,0 +1,29 @@
+
+/**
+ * @param {Object} [options = {}] - Options for the validation
+ * @param {RegExp} [options.regex =  /\n\s*[-*+–—]/] - Validation regex
+ * @param {String} [options.message = "This looks like a list, but it is plain text. Use the bulleted list option."] - Message to display when the validation triggers
+ */
+
+function noFakeLists(blocks, options = {}) {
+  const fakeListBlocks = (blocks || []).filter(
```

Comment:
I have the suspicion `blocks` will always be given as an array even when the editor is empty. We should double check that.